### DISCUSSION TITLE
xDrip Android Widget Font Size

### DISCUSSION THREAD

**Discussion by copacetic82** [NightscoutFoundation/xDrip#3032] [Ideas]
Hi

I currently use xDrip on an Android Tablet. I use the widget (on the home screen) to display my glucose level at night. Note that I am not talking about a screensaver or the Samsung always-on-display, but the actual home screen. 

Unfortunately the BG value is quite small for me, and I cant read it value without my glasses i.e. need to keep fiddling in the dark to find my glasses to read the value. Being able to increase the font size of the BG value would be of great help, but unfortunately I cant find an option to do this in the settings. Increasing the widget size does not increase the font size. 

Is this possible (and I am looking in the wrong place) or not? If not, I can imagine that this would be a feature helpful to a lot of other people.

Kind Regards
Ingo

**Comment by Navid200** [comment]
The title of your discussion has the word Widget in it.
The first paragraph states that you are not talking about screensaver, but, home screen.

Which one is it?

**Reply by copacetic82** [reply]
Hi...I am talking about the widget on the android home screen, see example below. I keep my Tablet "awake" with a black background, so the widget is always visible during the night. 

![image](https://github.com/NightscoutFoundation/xDrip/assets/45660618/be258266-a866-44f2-abf5-803e2e40cfb8)

**Comment by Navid200** [comment]
OK,

This sounds the same as this:
https://github.com/NightscoutFoundation/xDrip/issues/747
Please let me know if you disagree.

I will reopen that.

**Reply by copacetic82** [reply]
Oh thanks...did not find the other request earlier. Would be great if you could re-open it. :)
